includes/definitions/discovery/icotera.yaml

```diff
@@ -0,0 +1,52 @@
+mib: ICOTERA-I6400-SERIES
+modules:
+    sensors:
+        voltage:
+            options:
+                index: 0
+                divisor: 10000
+            data:
+                -
+                    oid: transceiverDdmVoltage
+                    num_oid: '.1.3.6.1.4.1.29865.11.3.1.3.4.{{ $index }}'
+                    index: 'transceiverDdmVoltage.{{ $index }}'
```

Comment:
You don't need to specify index unless you have conflicting indexes with other sensors.  It makes a sane index by default.

Reply:
No table exists in the MIB.

Reply:
That is irrelevant to my comment, please remove the index line.